test/pg_regress.sh

```diff
@@ -67,4 +67,23 @@ if [[ -z ${TESTS} ]] && [[ -z ${TEST_SCHEDULE} ]]; then
     exit 0;
 fi
 
+function cleanup() {
+  rm -rf ${EXE_DIR}/sql/dump
+  rm -rf ${TEST_TABLESPACE1_PATH}
+  rm -rf ${TEST_TABLESPACE2_PATH}
+  rm -f ${TEST_OUTPUT_DIR}/.pg_init
+}
+
+trap cleanup EXIT
+
+# This mktemp line will work on both OSX and GNU systems
+TEST_TABLESPACE1_PATH=${TEST_TABLESPACE1_PATH:-$(mktemp -d 2>/dev/null || mktemp -d -t 'timescaledb_regress')}
+TEST_TABLESPACE2_PATH=${TEST_TABLESPACE2_PATH:-$(mktemp -d 2>/dev/null || mktemp -d -t 'timescaledb_regress')}
```

Comment:
I think this will break on windows, and more generally `regresschecklocal`. Is it possible to do this in the `regresscheck` command instead?

Reply:
This is no change to current code and you can pass in any other path you setup outside of pg_regress.sh

Reply:
Are you trying to say that windows and regresschecklocal are currently broken?

Reply:
After further thought, I no longer think this will break it.
`regresschecklocal` is a bit weird, and requires more manual configuration, since `pg_regress.sh` potentially runs on a different machine than the run postgres is running. It means that these will be useless, but still not _wrong_.